snap/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: wsstat
+adopt-info: wsstat
+license: MIT
+summary: wsstat is a websocket traffic monitoring tool.
+description: |
+  wsstat provides a simple and easy to use tool to check the status and connection stats of a WebSocket endpoint.
+
+  Usage: wsstat wss://foo.example.com
+
+grade: stable
+confinement: strict
+base: core22
+
+apps:
+  wsstat:
+    command: bin/wsstat
+    plugs: [network, home]
+
+parts:
+  wsstat:
+    plugin: nil
+    build-snaps: 
+      - go
+    source: https://github.com/jakobilobi/wsstat.git
```

Comment:
```suggestion
    source: https://github.com/jakobilobi/wsstat.git
    # source-tag: v0.1.1  # uncomment this line to build on a specific tag instead of on the latest commit
```